plugins/calendar/src/main/resources/CalendarSpeech.aim.xml

```diff
@@ -96,35 +83,35 @@
 			</EntityTemplate>
 		</EntityTemplates>
 		<Prompt entityTemplateId="title">
-			<text>What is the title of the event?</text>
+			<text>How should I name it?</text>
 			<gram>{title}</gram>
 		</Prompt>		
 		<Prompt entityTemplateId="allday">
 			<text>Is it an all day event?</text>
 			<gram>{allday}</gram>
 		</Prompt>			
 		<Prompt entityTemplateId="start">
-			<text>When does the event start?</text>
+			<text>When does it start?</text>
 			<gram>[at] [the] {start}</gram>
 		</Prompt>		
 		<Prompt entityTemplateId="end">
-			<text>When does the event end?</text>
+			<text>When does it end?</text>
 			<gram>[at] [the] {end}</gram>
 		</Prompt>
 		<Prompt entityTemplateId="description">
-			<text>You can add a description to the event or just type no if you don't want to.</text>
+			<text>You can skip each of the following two questions by saying "no" after I asked them. What should I write in the description?</text>
```

Comment:
Be careful with this. The order of prompts amy asks might be changed in the future. 
For now it's all right. I might have to add a specific order attribute to the .aim.xml specification..

Reply:
Yes I know and considered taking the risk for now